vignettes/Accept-organism-for-GenomeInfoDb.Rmd

```diff
@@ -0,0 +1,60 @@
+---
+title: "Submitting your organism to GenomeInfoDb"
+author:
+- name: "Sonali Arora"
+- name: "H. Khan"
+  affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+package: GenomeInfoDb
+vignette: >
+  %\VignetteIndexEntry{Submitting your organism to GenomeInfoDb}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
```

Comment:
Small edit - Please change to include TOC using this code:

output:
  BiocStyle::html_document:
    number_sections: yes
    toc: yes
    toc_depth: 4